oteapi/models/resourceconfig.py

```diff
@@ -63,15 +64,12 @@ class ResourceConfig(BaseModel):
             "A rights statement that concerns how the distribution is accessed."
         ),
     )
-    description: Optional[str] = Field(
-        None, description="A free-text account of the distribution."
-    )
     publisher: Optional[str] = Field(
         None,
         description="The entity responsible for making the resource/item available.",
     )
-    configuration: Union[DataCacheConfig, Dict[str, Any]] = Field(
-        {},
+    configuration: DataCacheConfig = Field(
```

Comment:
When `DataCacheConfig` subclasses `AttrDict`, this makes sense. Otherwise it should be a `Union` of the two.

Reply:
`DataCacheConfig` did indirectly subclass `AttrDict` through `GenericConfig`, but now I've made the inheritance direct.